modules/cli/src/main/scala/scala/cli/commands/default/Default.scala

```diff
@@ -44,7 +44,8 @@ class Default(
 
   override def runCommand(options: DefaultOptions, args: RemainingArgs, logger: Logger): Unit =
     // can't fully re-parse and redirect to Version because of --cli-version and --scala-version clashing
-    if options.version then Version.runCommand(VersionOptions(options.shared.logging), args, logger)
+    if options.version then
+      Version.runCommand(VersionOptions(options.shared.logging, offline = true), args, logger)
```

Comment:
Instead of setting `offline`, we can ignore errors when something is wrong while downloading the latest tag of `scala-cli`.

Reply:
should be fixed.
i adjusted the PR description as well.